language-reference-manual/lrm.trlc

```diff
@@ -2,7 +2,7 @@ package LRM
 
 Versioning Version {
   major = 3
-  minor = 0
+  minor = 1
```

Comment:
Do we really need to increase the version for this patch?
I would argue that this should have been there with version 3.0 in the beginning.
So I would categorize this as a patch/fix for version 3.0

Reply:
Agreed. I have reverted the change.